core/server/data/migrations/versions/5.0/2022-05-11-13-12-settings-rename.js

```diff
@@ -0,0 +1,43 @@
+const logging = require('@tryghost/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+const renameMappings = [{
+    from: 'lang',
+    to: 'locale'
+}, {
+    from: 'session_secret',
+    to: 'admin_session_secret'
+}];
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        // eslint-disable-next-line no-restricted-syntax
+        for (const renameMapping of renameMappings) {
+            const oldSetting = await knex('settings')
+                .where('key', renameMapping.from)
+                .select('value')
+                .first();
+
+            if (!oldSetting) {
+                logging.warn(`Could not find setting ${renameMapping.from}, not updating ${renameMapping.to} value`);
+                continue;
+            }
+
+            const updatedValue = renameMapping.getToValue ? renameMapping.getToValue(oldSetting.value) : oldSetting.value;
```

Comment:
Given this isn't defined on any of the maps, and I don't think we have any requirement to map values as well as keys I think we can just do an `update settings set value = renameMapping.to where value = renameMapping.from` instead of doing an insert/delete?